### ISSUE TITLE
Error messages in log

### ISSUE THREAD

**Issue by ic-cedwards** [YodasMyDad/mvcforum#229]
I'm getting lots of error messages in my log like this:

> A post created activity record with id '9169bd78-8409-4363-bfb2-a88800e7d7a7' has a post id 'c3e082ba-cdf6-4633-95b2-a88800e7d765' that is not found in the post table. 

Generated by:

> MvcForum.Core.dll | ConvertToSpecificActivities | MvcForum.Core.Services.ActivityService

I get similar ones to do with topics, member joined, badges...  any idea what's causing them and what I can do to track down the issue please?

**Comment by YodasMyDad** [comment]
It just means a post has been deleted that is referenced in the activity table. I think I'll keep the log item, but delete the activity record to stop it showing.

**Comment by ic-cedwards** [comment]
Ah. That makes sense. Thanks for the update.  I was just getting worried about the log filling up rapidly.

Presumably it's the same for the topic, user & badge ones I'm getting then?

**Comment by YodasMyDad** [comment]
Yeah, anything like that, I should log then delete the activity record really.
